.npmignore

```diff
@@ -0,0 +1,32 @@
+Makefile
+CmakeLists.txt
+*yml
+build
+cmake
+misc
+.*
+*tgz
+benchmark
+expression-test
+cloudformation
+bin
+vendor
+include
+mason_packages
+mapbox-gl-js
+test
+lib
+next
+src
+scripts
+render-test
+platform/android
+platform/darwin
+platform/default
+platform/glfw
+platform/ios
+platform/linux
+platform/macos
+platform/qt
+platform/node/symbol-list
+platform/node/version-script
```

Comment:
Another option I considered to reduce package size while avoiding having to declare all these things is to move the `package.json` into `platform/node/` and do the packaging from there. But that would be more distruptive of a change, so I did not take that route.

Also, I wish there was such a thing as `.npminclude` but it does not exist: https://github.com/npm/npm/issues/13171